R/selections.R

```diff
@@ -221,13 +221,26 @@ recipes_eval_select <- function(quos, data, info, ..., allow_rename = FALSE,
 #' `has_role()`, `all_predictors()`, and `all_outcomes()` can be used to
 #'  select variables in a formula that have certain roles.
 #'
-#' Similarly, `has_type()`, `all_numeric()`, and `all_nominal()` are used to
-#'  select columns based on their data type. Nominal variables include both
-#'  character and factor.
+#'  **In most cases**, the right approach for users will be use to use the
+#'  predictor-specific selectors such as `all_numeric_predictors()` and
+#'  `all_nominal_predictors()`. In general you should be careful about using
+#'  `-all_outcomes()` if a `*_predictors()` selector would do what you want.
 #'
-#' **In most cases**, the selectors `all_numeric_predictors()` and
-#'  `all_nominal_predictors()`, which select on role and type, will be the right
-#'  approach for users.
+#'  Similarly, `has_type()`, `all_numeric()`, `all_integer()`, `all_double()`,
+#'  `all_nominal()`, `all_ordered()`, `all_unordered()`, `all_factor()`,
+#'  `all_string()`, `all_date()` and `all_datetime()` are used to select columns
+#'  based on their data type.
+#'
+#'  `all_factor()` captures unordered factors, `all_string()` captures
```

Comment:
I think that `all_factor()` should get both ordered and unordered.

Reply:
it actually did that, just wasn't reflected in the documentation https://github.com/tidymodels/recipes/pull/993/commits/e7814ac94c9440958d085eed77700567751d0a74